code/modules/clothing/suits/armor.dm

```diff
@@ -476,6 +476,24 @@
 	siemens_coefficient = 0
 	price_tag = 600
 	//Used ablative gear armor values and technomancer helmet/voidsuit values.
+//onestar
```

Comment:
Could skip a line above for better readability, like you did in `code/modules/clothing/under/miscellaneous.dm`

Reply:
Thanks for the suggestion.